<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="face_recognition_node" pkg="face_recognition" type="face_recognition_node.py" output="screen"/>

  <node name="classifier_training_node" pkg="face_recognition" type="classifier_training_node.py" output="screen"/>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/vision_system/fr/face_recognition_view"/>
    <param name="autosize" value="true" />
  </node>

</launch>
